installation/docker-containers/docker-swarm/README.md

```diff
@@ -0,0 +1,22 @@
+This is the easiest way to setup RocketChat on Docker Swarm:
+
+  1. Make sure your Cloudflare dashboard shows correct DNS entries for `traefik.${DOMAIN_WITH_DNS_HANDLED_BY_CLOUDFLARE}` and `rocketchat.${DOMAIN_WITH_DNS_HANDLED_BY_CLOUDFLARE}` pointing to your docker swarm.
```

Comment:
This should be tweaked.  We should not be assuming that someone is using cloudflare for dns.  Using CLOUDFLARE through out the whole document is just going to lead to confusion